migrations/migration1.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE action
```

Comment:
Info: any reason we are implementing this as a migration?
If we switch to an ORM, we'll need to be careful with how we treat theses changes

Reply:
Since the tables are already deploy, I can only alter the tables. 
We can change later depending on the ORM.